my-code/movie-search/src/Containers/SearchBar/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchMovies } from '../../Actions';
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { searchTerm: '' };
+  }
+
+  onInputChange(searchTerm) {
+    this.setState({ searchTerm });
+    this.props.fetchMovies(searchTerm);
```

Comment:
We're making A LOT of requests here!
consider throttling or debouncing these requests

Reply:
Right. I wasn't able to add the debounce on this one, my bad.